session/session.go

```diff
@@ -660,6 +660,10 @@ func (s *session) retry(ctx context.Context, maxCnt uint) (err error) {
 		if err != nil {
 			return err
 		}
+		pessTxnConf := config.GetGlobalConfig().PessimisticTxn
```

Comment:
It will retry all txn in pessimistic mode, which is different with before. It will only retry autocommit txn in pessimistic mode.